### PR TITLE
Main program

### PR DESCRIPTION
**Pull request by rarmstrong3**

### CODE REVIEW

**Comment by nuclearkatie** [comment]
I kind of hate to be this person at the very end of the semester but there are many small (and quick!) changes that could be made so this program conforms to PEP8

**Comment by rarmstrong3** [comment]
Unfortunately, I will not have any time to fix those before class today. If someone has time, I believe they could clone my fork of the repository to fix these errors.

**Comment by Zach-Fiscus** [comment]
I should have forked your repository before making the changes and pulled request to your repository, but I made the pep8 changes and the commits were merged into your pull request automatically. Whoops.

**Comment by nuclearkatie** [comment]
Thanks @Zach-Fiscus! I'll fix the others and push them here (I don't think it requires a pull request into this branch, I'm going to do the same thing you did)

**Comment by rarmstrong3** [comment]
Awesome! Thanks @Zach-Fiscus  @nuclearkatie

File: src/transient_plot.py [hunk 1]
```diff
@@ -1,36 +1,38 @@
...
+    the materials assigned to them
+
     output
     ------
     A plot of the temperature at each node, at each timestep
     """
-    
+
     mesh_radii = []
     counter = 0
```
Comment (Zach-Fiscus) [inline src/transient_plot.py:22]: This counter is an unused variable.

Comment (nuclearkatie) [reply src/transient_plot.py:22]: Good catch @Zach-Fiscus

**Review by Zach-Fiscus — APPROVED** [review]
Looks good, thanks for writing this up @rarmstrong3, and for polishing it up a bit @nuclearkatie. Great end to a great project.

**Pull request merged**